docs/core/encoding.md

```diff
@@ -97,10 +97,10 @@ Modules are encouraged to utilize Protobuf encoding for their respective types.
 
 In addition to [following official Protocol Buffer guidelines](https://developers.google.com/protocol-buffers/docs/proto3#simple), we recommend using these annotations in .proto files when dealing with interfaces:
 
-* use `cosmos_proto.accepts_interface` to annote fields that accept interfaces
-* pass the same fully qualified name as `protoName` to `InterfaceRegistry.RegisterInterface`
-* annotate interface implementations with `cosmos_proto.implements_interface`
-* pass the same fully qualified name as `protoName` to `InterfaceRegistry.RegisterInterface`
+- use `cosmos_proto.accepts_interface` to annote fields that accept interfaces
```

Comment:
Let's not revert that. Lists should use '*': https://github.com/cosmos/cosmos-sdk/blob/main/.markdownlint.json

You can install a markdown lint extension in your editor for enforcing that those rules are being followed.

Reply:
Installed and done!